README.md

```diff
@@ -66,11 +68,37 @@ hello_ex1:
   shell: "echo hello"
 ```
 
+Note that the outer structure (the version and general yaml file with named sections)
```

Comment:
I think for end-users it would be better if there is a table to graphically represent the outer keys. For example the ``version`` key is required. Along with name of sub-test. I call it **subtest** instead of **test** to differentiate the actual YAML file or **test-configuration** and actual generated script or **testscript**. It's important to define these terminologies either in buildtest docs or in Schema so we use the correct terminology. 

I think a table something along the lines of https://buildtest.readthedocs.io/en/latest/build_subcommand/singlesource_schema.html would be required for each schema.

Reply:
This is a great idea! I'll add tables to the markdown here for now, and we can eventually render that into a GitHub pages site (any updates on the org name?)

Reply:
no updates on org yet. Let me reach out to all the committers in https://github.com/buildtest and see if one of them can reply. If not i will see what GitHub can do in a few days. Otherwise let's plan to use this org for now and we can move everything later. It's just extra overhead moving everything now as it would break alot of links

Reply:
okay tables are added.